pkgs/servers/isso/default.nix

```diff
@@ -33,16 +29,21 @@ with python3Packages; buildPythonApplication rec {
     werkzeug
     bleach
     flask-caching
+    sphinx
```

Comment:
If this is used by the sphinxHook, it goes into nativeBuildInputs, too

Reply:
right